### DISCUSSION TITLE
Dot instead of semicolon as a separator in figure and table captions

### DISCUSSION THREAD

**Discussion by robertoragno** [quarto-dev/quarto-cli#10913] [Q&A] [answered]
### Description

Hello,

I am using Quarto to generate the PDF of a book. If I use:

```{r}
#| fig-cap: "Speed and Stopping Distances of Cars"
#| label: fig-cars

plot(cars)
```

The generated figure label is: "Figure 1: Speed and Stopping Distances of Cars". Is there any way to change the semicolon to a period to obtain "Figure 1. Speed and Stopping Distances of Cars"?

Thank you in advance for your help!

**Comment by robertoragno** [comment, marked as answer]
Solved with:

```
header-includes:
   \usepackage{caption}
   \captionsetup{labelsep=period}
```

In the YAML

**Reply by mcanouil** [reply]
FYI, this is a Pandoc option. The Quarto option is `include-in-header`, see documentation.

**Comment by mcanouil** [comment]
Have you considered using `title-delim`, as described in <https://quarto.org/docs/authoring/cross-reference-options.html>?

**Reply by robertoragno** [reply]
I did not notice this! This is great to know. Thanks so much!

**Comment by nithinmkp** [comment]
As a follow up question here, how can we make Figure (1),  Table (1) etc? also is there a way to achieve define Figure (1) where were need capital letter and figure (1) when we need small letters? in latex we can use cleveref package

**Reply by mcanouil** [reply]
For short, no.
